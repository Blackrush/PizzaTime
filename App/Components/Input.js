import React, { Component } from 'react'
import {
  StyleSheet,
  View,
  TextInput,
} from 'react-native'


class Input extends Component {
  render() {
    return (
      <View style={styles.inputView}>
        <TextInput style={styles.input} {...this.props}/>
      </View>
    )
  }
}


const styles = StyleSheet.create({
  inputView: {
    flex: 1,
    flexDirection: 'row',

    margin: 3,
    paddingLeft: 10,
    paddingRight: 10,

    borderBottomWidth: .5,
    borderBottomColor: '#8D8D92',
  },
  input: {
    flex: 1,
    height: 39,
  },
})


export default Input