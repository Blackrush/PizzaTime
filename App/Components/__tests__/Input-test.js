import React from 'react'
import Input from '../Input'
import { create as render } from 'react-test-renderer'

jest.mock('TextInput', () => 'TextInput')

test('Input renders correctly', () => {
  const tree = render(
    <Input value="test"/>
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
