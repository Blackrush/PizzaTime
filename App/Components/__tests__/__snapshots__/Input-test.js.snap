exports[`test Input renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#8D8D92",
      "borderBottomWidth": 0.5,
      "flex": 1,
      "flexDirection": "row",
      "margin": 3,
      "paddingLeft": 10,
      "paddingRight": 10
    }
  }>
  <TextInput
    style={
      Object {
        "flex": 1,
        "height": 39
      }
    }
    value="test" />
</View>
`;
